src/qt/trafficgraphwidget.h

```diff
@@ -22,22 +24,22 @@ class TrafficGraphWidget : public QWidget
 public:
     explicit TrafficGraphWidget(QWidget *parent = nullptr);
     void setClientModel(ClientModel *model);
-    int getGraphRangeMins() const;
+    std::chrono::minutes getGraphRange() const;
```

Comment:
Just remove? It's not used.

Reply:
@promag @shaavan 

I concur - It appears to be part of the original implementation but never used for anything...

it can and should be removed...

https://github.com/bitcoin-core/gui/blame/16781e1bc9f8ffc721ebea73434e0066957bc959/src/qt/trafficgraphwidget.cpp#L45


![Screen Shot 2022-01-20 at 1 16 34 PM](https://user-images.githubusercontent.com/152159/150397764-0f360acc-dd05-46b1-9bb1-371e2bb76617.png)

Reply:
@promag - great catch!

Reply:
Thanks @promag for catching this. Addressed in 5efd391eb1ea1a3551b2335827e4b5577be2aca3

Reply:
> Approach ACK Removing the TrafficGraphWidget::getGraphRangeMins() function nACK
> 
> I use int TrafficGraphWidget::getGraphRangeMins() const in PR #539 (which fixes issue #532)
> 
> I have added a variation to of this PR to #539 [c25d3f4](https://github.com/bitcoin-core/gui/commit/c25d3f42084926c92ae2051278cd785cb1907d56) to ensure compatibility with the chrono lib.
> 
> https://github.com/bitcoin-core/gui/blob/c25d3f42084926c92ae2051278cd785cb1907d56/src/qt/trafficgraphwidget.cpp#L46
> 
> ### Note
> https://github.com/bitcoin-core/gui/pull/524/files#diff-da38549cd85e0c422891b8ef1eb3cc33e76f0b80d15ca63aa7ad6f065c8ba694L44